backend/app/migrator/mapper.go

```diff
@@ -37,7 +37,7 @@ func (u *URLMapper) loadRules(reader io.Reader) error {
 
 	rulesText := strings.TrimSpace(string(data))
 
-	u.rules = make(map[string]string)
+	u.rules = map[string]string{}
```

Comment:
this one I don't really get either. make seems like a clear intention here and for a potentially dangerous thing like make I'd prefer make way